app/views/latest/index.html.erb

```diff
@@ -1,71 +1,79 @@
-<% page_title t("document.latest_from", subject: subject) %>
+<% page_title t("document.latest") %>
 <% page_class "documents-index" %>
 <% atom_discovery_link_tag filter_atom_feed_url, "Recent documents" %>
 
-<header class="block headings-block">
-  <div class="govuk-width-container">
-    <% if  subject.is_a? Organisation %>
-      <%= render "govuk_publishing_components/components/breadcrumbs", {
-        breadcrumbs: [
-          {
-            title: "Home",
-            url: "/",
-          },
-          {
-            title: "Organisations",
-            url: "/government/organisations",
-          },
-          {
-            title: "#{subject}",
-            url: "/government/organisations/#{subject['slug']}",
-          },
-        ]
-      } %>
-    <% elsif subject.is_a? TopicalEvent %>
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <% if  subject.is_a? Organisation %>
         <%= render "govuk_publishing_components/components/breadcrumbs", {
           breadcrumbs: [
             {
               title: "Home",
               url: "/",
             },
             {
-              title: "Topical events",
-              url: "/government/topical-events",
-            },
-            {
-              title: "#{subject}",
-              url: "/government/topical-events/#{subject['slug']}",
-            },
-          ]
-        } %>
-    <% else subject.is_a? WorldLocation %>
-      <%= render "govuk_publishing_components/components/breadcrumbs", {
-          breadcrumbs: [
-            {
-              title: "Home",
-              url: "/",
-            },
-            {
-              title: "Help and services around the world",
-              url: "/government/world",
+              title: "Organisations",
+              url: "/government/organisations",
             },
             {
               title: "#{subject}",
-              url: "/government/world/#{subject['slug']}",
+              url: "/government/organisations/#{subject['slug']}",
             },
           ]
         } %>
-    <% end %>
-    <%= render "govuk_publishing_components/components/title", {
-      context: subject,
-      title: t("document.latest"),
-    } %>
+      <% elsif subject.is_a? TopicalEvent %>
+          <%= render "govuk_publishing_components/components/breadcrumbs", {
+            breadcrumbs: [
+              {
+                title: "Home",
+                url: "/",
+              },
+              {
+                title: "Topical events",
+                url: "/government/topical-events",
+              },
+              {
+                title: "#{subject}",
+                url: "/government/topical-events/#{subject['slug']}",
+              },
+            ]
+          } %>
+      <% else subject.is_a? WorldLocation %>
```

Comment:
I might be wrong... but I think `else` if correct? The structure of this block seems to be -
```
    <% if subject.is_a? Organisation %>...
    <% elsif subject.is_a? TopicalEvent %>...
    <% else subject.is_a? WorldLocation %>...    
    <% end %>
```
So, it looks like the breadcrumb changes based on the subject, since it's a shared page, with "World Location" the default subject.